sig/prototype/app/helpers/sessions_helper.rbs

```diff
@@ -0,0 +1,7 @@
+module SessionsHelper
+  @current_user: untyped
+
+  def current_user: () -> (untyped | nil)
```

Comment:
`#current_user` and `#logged_in?` are often used in this application. Therefore it's better to give types to this module instead of auto-generation. But I prefer to give types to the whole of the application first. So I'll do it in the following step.